backend/battle/templates/battle/pokemon_form.html

```diff
@@ -21,12 +21,21 @@
                     <p>
                         {{ form.errors.pokemon_1 }}
                         <label for="id_pokemon_1">Pokemon 1:</label>
-                        <input id="pokemon_1" name="pokemon_1" type="text" list="pokemons">
-                        <datalist id="pokemons" name="pokemon_1">
-                            {% for results in pokemons %}
-                                <option value="{{results.name}}">
-                            {% endfor %}
-                        </datalist>
+                        {% if pokemon_1 %}
+                            <input id="pokemon_1" name="pokemon_1" type="text" list="pokemons" value={{pokemon_1}}>
+                            <datalist id="pokemons" name="pokemon_1">
+                                {% for results in pokemons %}
+                                    <option value="{{results.name}}">
```

Comment:
No closing tag for `option`, is this expected?

Reply:
Yup. Because the value of the option I defined on the tag value

Reply:
Thank you :)